### PR TITLE
Make upcoming events only in the events list

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

**Comment by hhamon** [comment]
This is still WIP for now.

File: src/AppBundle/Repository/EventRepository.php [hunk 1]
```diff
@@ -182,7 +182,7 @@ public function searchEvents(SearchParametersFilter $search): array
                 ->andWhere($this->getNearbyExpression().' < :distance_max')
                 ->andWhere('n.beginAt > :now')
                 ->setParameter('distance_max', $search->getRadius())
-                ->setParameter('now', new \DateTime('-36 hours'))
```
Comment (mykiwi) [inline src/AppBundle/Repository/EventRepository.php]: Why remove this?

Comment (tgalopin) [inline src/AppBundle/Repository/EventRepository.php]: It's the issue, that was a misexplaination in the US

Comment (hhamon) [inline src/AppBundle/Repository/EventRepository.php]: To fix the problem!

File: src/AppBundle/Controller/EventController.php [hunk 2]
```diff
@@ -61,6 +61,10 @@ public function exportIcalAction(Event $event): Response
      */
     public function attendAction(Request $request, Event $event): Response
     {
+        if ($event->isFinished()) {
+            throw $this->createNotFoundException(sprintf('Event "%s" is finished and does not accept registrations anymore', $event->getUuid()));
```
Comment (rpg600) [inline src/AppBundle/Controller/EventController.php]: Duno if it's relevant to do a functional test for this case.

Comment (tgalopin) [inline src/AppBundle/Controller/EventController.php]: Let's do this in a new PR.

**Pull request merged**